scripts/fuzz_opt.py

```diff
@@ -1680,7 +1680,8 @@ def get_random_opts():
 # some features depend on other features, so if a required feature is
 # disabled, its dependent features need to be disabled as well.
 IMPLIED_FEATURE_OPTS = {
-    '--disable-reference-types': ['--disable-gc'],
+    '--disable-reference-types': ['--disable-gc', '--disable-strings'],
+    '--disable-gc': ['--disable-strings'],
```

Comment:
This may actually be needed on `main` now that the previous PR landed - should have perhaps been included there just to be safe. If this PR can't land quickly then I'll split it out.